docs/001_develop/03_client-capabilities/008_interaction/criteria-segmented-control.mdx

```diff
@@ -0,0 +1,241 @@
+          `,
+      })
+      export class MyElement extends GenesisElement {
+        const criteriaFilter: string;
+        const toolbarOptions: CriteriaSegmentedControlOption[] = [
+          { label: 'A', field: 'CHOSEN_OPTION', value: 'option-A', serialiser: Serialisers.EQ },
+          { label: 'B', field: 'CHOSEN_OPTION', value: 'option-B', serialiser: Serialisers.EQ },
+          { label: 'C', field: 'CHOSEN_OPTION', value: 'option-C', serialiser: Serialisers.EQ },
+        ];
+
+        onChange() {
+          console.log(this.criteriaFilter)
+        }
+      }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+    Declaration
+    ```html
+      <criteria-segmented-control></criteria-segmented-control>
+    ```
+    Usage
+    ```typescript
+      export function MyComponent() {
+
+        const criteriaOptions = [
+          { label: 'A', field: 'CHOSEN_OPTION', value: 'option-A', serialiser: Serialisers.EQ },
+          { label: 'B', field: 'CHOSEN_OPTION', value: 'option-B', serialiser: Serialisers.EQ },
+          { label: 'C', field: 'CHOSEN_OPTION', value: 'option-C', serialiser: Serialisers.EQ },
+        ];
+
+        const handleCriteriaChange = (event) => {
+          console.log(event.target.value)
+        }
+
+        return (
+          <criteria-segmented-control criteriaOptions={{criteriaOptions}} onClick={handleCriteriaChange}>
+            <label slot="label">Select option</label>
+          </criteria-segmented-control>
+
+        );
+      }
+    ```
+  </TabItem>
+  <TabItem value="angular">
+
+    Declaration
+    ```html
+      <criteria-segmented-control></criteria-segmented-control>
+    ```
+
+    Usage
+    ```typescript
+      import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+      import { FormsModule } from '@angular/forms';
+      @Component({
+        selector: 'my-root',
+        template: `
+              <criteria-segmented-control
+                [criteriaOptions]="criteriaOptions"
+                (change)="handleCriteriaChange($event)">
+                <label slot="label">Filter by instrument name</label>
+              </criteria-segmented-control>
+            `,
+        standalone: true,
+        schemas: [CUSTOM_ELEMENTS_SCHEMA],
+        imports: [FormsModule],
+      })
+      export class AppComponent {
+
+        const criteriaOptions = [
+          { label: 'A', field: 'CHOSEN_OPTION', value: 'option-A', serialiser: Serialisers.EQ },
+          { label: 'B', field: 'CHOSEN_OPTION', value: 'option-B', serialiser: Serialisers.EQ },
+          { label: 'C', field: 'CHOSEN_OPTION', value: 'option-C', serialiser: Serialisers.EQ },
+        ];
+
+        handleCriteriaChange(event: Event) {
+          const target = event.target as HTMLInputElement;
+          console.log(target.value);
+        }
+      }
+      ```
+    </TabItem>
+  </Tabs>
+
+Criteria options are JSON objecst constructed using serialisers
+
+The choice of serialisers are:
+
+```typescript
```

Comment:
What these stand for should be obvious to many, but do you think it's worth elaborating on them just in case? e.g. GE = Greater than or equal to. Happy if you think not useful

Reply:
I was a bit wary on this, as I feel like it should really be a link to 
https://learn.genesis.global/docs/web/basics/criteria-matching/

Reply:
Makes sense to do that. Maybe just add a TODO comment saying we need to do that